Shared/Flows/Photo Capture/ExpressionCaptureViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ExpressionCaptureViewController.swift
+//  Jibber
+//
+//  Created by Martin Young on 4/28/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class ExpressionPhotoCaptureViewController: ViewController {
```

Comment:
This was different enough from the profile image controller that I made it separate.